hotshot-state-prover/src/circuit.rs

```diff
@@ -68,10 +66,6 @@ impl LightClientStateVar {
                 circuit.create_public_variable(view_number_f)?,
                 circuit.create_public_variable(block_height_f)?,
                 circuit.create_public_variable(state.block_comm_root)?,
-                circuit.create_public_variable(state.fee_ledger_comm)?,
-                circuit.create_public_variable(state.stake_table_comm.0)?,
-                circuit.create_public_variable(state.stake_table_comm.1)?,
-                circuit.create_public_variable(state.stake_table_comm.2)?,
```

Comment:
these changes are wrong. 

we have correctly update these part in https://github.com/EspressoSystems/espresso-sequencer/pull/1939

should we consider pointing to latest tag of hotshot in that PR instead?
otherwise, we need to cherry-pick many changes from that PR.